### ISSUE TITLE
Cannot install the Package since recently

### ISSUE THREAD

**Issue by Rouux** [h8man/NavMeshPlus#140]
I used this package (it's amazing btw, it real does the job for most of the stuffs I need in 2D considering path finding)
I created a new project and tried installing NewMeshPlus but I just have a few errors and the install is never really done.

Looking at the errors I get it seems to be linked to one the last pull request merged which was https://github.com/h8man/NavMeshPlus/pull/139 :
```shell
<userpath>/Downloads\NavMeshPlus-master\NavMeshComponents\Scripts\NavMeshBuilder2d.cs(129,17): error CS0103: The name 'map' does not exist in the current context
<userpath>/Downloads\NavMeshPlus-master\NavMeshComponents\Scripts\NavMeshBuilder2d.cs(130,17): error CS0103: The name 'coliderMap' does not exist in the current context
```

Which results in a big error
```shell
Failed to find entry-points:
Mono.Cecil.AssemblyResolutionException: Failed to resolve assembly: 'NavMeshPlus, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null'
[...]
While compiling job
```

Failing to install the package due to compilation error it seems

Do you have any idea if there's really an issue there or if I did something wrong ? (I tried by adding the package by both git url and package.json file)

To reproduce you can try to install NaxMeshPlus on any new 2D project (I created a projet using the 2D (URP) preset but I believe that has no effect on the outcome)

**Comment by Rouux** [comment]
I do not see how it resolves the issue mentioned in the first place by #139 but I opened a pull request believing to fix what's wrong as #141 
I do not know if it was the behavior initially intended
Using my fork code (so the changes in the PR) I can install NewMeshPlus and use its components without anymore issues.

**Comment by h8man** [comment]
Thx, renaming fail. No time to test(
